File: v2/csharp/src/main/csharp/vestorly/Model/MemberReport.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class MemberReport {
    
    
    [DataMember(Name="day", EmitDefaultValue=false)]
    public int? day { get; set; }

    
    
    [DataMember(Name="event_count", EmitDefaultValue=false)]
    public int? eventCount { get; set; }

    
    
    [DataMember(Name="originator_group_id", EmitDefaultValue=false)]
    public string originatorGroupId { get; set; }

    
    
    [DataMember(Name="parent_originator_id", EmitDefaultValue=false)]
    public string parentOriginatorId { get; set; }

    
    
    [DataMember(Name="source", EmitDefaultValue=false)]
    public string source { get; set; }

    
    
    [DataMember(Name="type", EmitDefaultValue=false)]
    public string type { get; set; }

    
    
    [DataMember(Name="year", EmitDefaultValue=false)]
    public int? year { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MemberReport {\n");
      
      sb.Append("  day: ").Append(day).Append("\n");
      
      sb.Append("  eventCount: ").Append(eventCount).Append("\n");
      
      sb.Append("  originatorGroupId: ").Append(originatorGroupId).Append("\n");
      
      sb.Append("  parentOriginatorId: ").Append(parentOriginatorId).Append("\n");
      
      sb.Append("  source: ").Append(source).Append("\n");
      
      sb.Append("  type: ").Append(type).Append("\n");
      
      sb.Append("  year: ").Append(year).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/MemberReport.java
package vestorly.client.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class MemberReport  {
  
  private Integer day = null;
  private Integer eventCount = null;
  private String originatorGroupId = null;
  private String parentOriginatorId = null;
  private String source = null;
  private String type = null;
  private Integer year = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("day")
  public Integer getDay() {
    return day;
  }
  public void setDay(Integer day) {
    this.day = day;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("event_count")
  public Integer getEventCount() {
    return eventCount;
  }
  public void setEventCount(Integer eventCount) {
    this.eventCount = eventCount;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("originator_group_id")
  public String getOriginatorGroupId() {
    return originatorGroupId;
  }
  public void setOriginatorGroupId(String originatorGroupId) {
    this.originatorGroupId = originatorGroupId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("parent_originator_id")
  public String getParentOriginatorId() {
    return parentOriginatorId;
  }
  public void setParentOriginatorId(String parentOriginatorId) {
    this.parentOriginatorId = parentOriginatorId;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("source")
  public String getSource() {
    return source;
  }
  public void setSource(String source) {
    this.source = source;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("type")
  public String getType() {
    return type;
  }
  public void setType(String type) {
    this.type = type;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("year")
  public Integer getYear() {
    return year;
  }
  public void setYear(Integer year) {
    this.year = year;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class MemberReport {\n");
    
    sb.append("  day: ").append(day).append("\n");
    sb.append("  eventCount: ").append(eventCount).append("\n");
    sb.append("  originatorGroupId: ").append(originatorGroupId).append("\n");
    sb.append("  parentOriginatorId: ").append(parentOriginatorId).append("\n");
    sb.append("  source: ").append(source).append("\n");
    sb.append("  type: ").append(type).append("\n");
    sb.append("  year: ").append(year).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
